recirq/quantum_chess/quantum_board.py

```diff
@@ -27,12 +27,16 @@
     xy_to_bit,
     bit_ones,
 )
+from recirq.quantum_chess.caching_utils import CacheKey, cache_key_from_move
 import recirq.quantum_chess.circuit_transformer as ct
 import recirq.quantum_chess.enums as enums
 import recirq.quantum_chess.move as move
 import recirq.quantum_chess.quantum_moves as qm
 
 
+# This is the basis state corresponding to FEN: rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR.
+DEFAULT_CHESS_INIT_STATE = 18446462598732906495
```

Comment:
Hex could be more readable, `0xffff00000000ffff`

Reply:
Done